device/pcie/pci_device.cpp

```diff
@@ -258,7 +258,10 @@ PCIDevice::PCIDevice(int pci_device_number, int logical_device_id) :
     numa_node(read_sysfs<int>(info, "numa_node")),
     revision(read_sysfs<int>(info, "revision")),
     arch(detect_arch(info.device_id, revision)),
-    architecture_implementation(tt::umd::architecture_implementation::create(arch)) {
+    architecture_implementation(tt::umd::architecture_implementation::create(arch)),
+    kmd_version(read_kmd_version()) {
+    log_info(LogSiliconDriver, "Opened device {}; KMD version: {}", pci_device_num, kmd_version.to_string());
```

Comment:
nit: add PCI to message: Opened PCI device 

I think it would make less confusion (given that this is log info) for end users.

Reply:
Added